conf/nomad/presto.hcl

```diff
@@ -385,6 +385,7 @@ EOF
       }
       resources {
         memory = ${memory}
+        cpu = ${cpu}
```

Comment:
```suggestion
        memory = ${memory}
        cpu    = ${cpu}
```
# :shrug:

Reply:
I wonder does `make pre-commit` help in that context. [link to command](https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/master/Makefile#L100)?

Reply:
Huh, good point. I'll test it out now!

Reply:
> I wonder does `make pre-commit` help in that context. [link to command](https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/master/Makefile#L100)?

Yes, that should do the trick!